test/ggsn_proxy_SUITE.erl

```diff
@@ -619,7 +619,8 @@ common() ->
      session_accounting,
      sx_upf_reconnect,
      sx_upf_removal,
-     sx_timeout
+     sx_timeout,
+     delete_bearer_requests_multi
```

Comment:
I was try to keep one name for test cases, but looks like the name `delete_pdp_context_requests_multi` instead of `delete_bearer_requests_multi` for test case in `test/ggsn_proxy_SUITE.erl` can be more suit.